events/2018/sponsors.md

```diff
@@ -9,78 +9,48 @@ Sponsors contribute the funds and resources that allow us to run the entire conf
-</tr>
-<tr style="height: 64px;">
-<th style="text-align: center; height: 64px; width: 99%;" colspan="2">
-<h1><strong>Gold Sponsors</strong></h1>
-</th>
-</tr>
-<tr style="height: 21px;">
-<td style="height: 21px; width: 50%;"><img src="/images/2018/web-logos/cyberark.png" alt="" style="display:block;margin:0 auto" /></td>
-<td style="height: 21px; width: 50%;"><img src="/images/2018/web-logos/dejavu.png" alt="" style="display:block;margin:0 auto" /></td>
-</tr>
-<tr style="height: 21px;">
-<td style="height: 21px; width: 50%;"><img src="/images/2018/web-logos/eclypsium.png" alt="" style="display:block;margin:0 auto" /></td>
-<td style="height: 21px; width: 50%;"><img src="/images/2018/web-logos/fortinet.png" alt="" style="display:block;margin:0 auto" /></td>
-</tr>
-<tr style="height: 21px;">
-<td style="height: 21px; width: 50%;"><img src="/images/2018/web-logos/bpm.png" alt="" style="display:block;margin:0 auto" /></td>
-<td style="height: 21px; width: 50%;"><img src="/images/2018/web-logos/incomm.png" alt="" style="display:block;margin:0 auto" /></td>
-</tr>
-<tr style="height: 21px;">
-<td style="height: 21px; width: 50%;"><img src="/images/2018/web-logos/newrelic.png" alt="" style="display:block;margin:0 auto" /></td>
-<td style="height: 21px; width: 50%;"><img src="/images/2018/web-logos/recordedfuture.png" alt="" style="display:block;margin:0 auto" /></td>
-</tr>
-<tr style="height: 21px;">
-<td style="height: 21px; width: 50%;"><img src="/images/2018/web-logos/tripwire.png" alt="" style="display:block;margin:0 auto" /></td>
-<td style="height: 21px; width: 50%;"><img src="/images/2018/web-logos/twilio.png" alt="" style="display:block;margin:0 auto" /></td>
-</tr>
-<tr style="height: 64px;">
-<th style="text-align: center; height: 64px; width: 99%;" colspan="2">
-<h1><strong>Silver Sponsors</strong></h1>
-</th>
-</tr>
-<tr style="height: 21px;">
-<td style="height: 21px; width: 50%;"><img src="/images/2018/web-logos/identitytechnologies.png" alt="" style="display:block;margin:0 auto" /></td>
-<td style="height: 21px; width: 50%;"><img src="/images/2018/web-logos/isaca.png" alt="" style="display:block;margin:0 auto" /></td>
-</tr>
-<tr style="height: 21px;">
-<td style="height: 21px; width: 50%;"><img src="/images/2018/web-logos/nostarch.png" alt="" style="display:block;margin:0 auto" /></td>
-<td style="height: 21px; width: 50%;"><img src="/images/2018/web-logos/summit.png" alt="" style="display:block;margin:0 auto" /></td>
-</tr>
-<tr style="height: 64px;">
-<th style="text-align: center; height: 64px; width: 99%;" colspan="2"></th></tr>
-</tbody>
-</table>
+<div class="row">
+  <div class="columns small-12"><h1 class="center-text"><strong>Platinum Sponsors</strong></h1></div>
+</div>
+<p>&nbsp;</p>
+<div class="row">
+  <div class="columns small-6"><img src="/images/2018/web-logos/a10.png" alt="" class="center"/></div>
+  <div class="columns small-6"><img src="/images/2018/web-logos/ctrlh.png" alt="" class="center"/></div>
```

Comment:
Each row has to add 12 columns. This can be changed into 'small-6 large-4' to make it render differently in large/small screens. Having it done this way makes the job of adding or removing one logo much easier, but the class 'end' would have to be added to the last item of each grid to prevent it from rendering on the right leaving blank spaces.